full/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN curl -o /var/lib/apt/dazzle-marks/llvm.gpg -fsSL https://apt.llvm.org/llvm-s
         clang-format \
         clang-tidy \
         gdb \
-        lld
+        lld ripgrep
```

Comment:
Semantically, I'm not sure the C/C++ section is the most appropriate to install `ripgrep` in. Maybe it should be installed in `base/Dockerfile` instead?

Also, please at least keep one package per line, and sort packages alphabetically. 😇

Reply:
Ah well. I am setting up in base/Dockerfile instead